tests/sqllogictests/suites/tpch_iceberg/utils.test

```diff
@@ -105,8 +105,11 @@ id INT YES NULL (empty)
 statement ok
 create or replace table abc.t1(c1 int, c2 string);
 
+statement error 1001
+create or replace table abc.t1(c1 datetime, c2 string) partition by (c1, c2);
```

Comment:
If we apply the partition by on the fuse engine table, what's happened? By design, it will return not support error. For the user, they don't know the table is fuse or iceberg, but the capability is not same. We should handle them on the underlying.

Reply:
Oh yes I will add this check

Reply:
Now if create a table in default/hive catalog with a partition or PROPERTIES it will return a definite err.